purl-dev/schema.xml

```diff
@@ -14,6 +14,14 @@
 
     <field name="id" type="string" stored="true" indexed="true" multiValued="false" required="true"/>
     <field name="timestamp" type="date" indexed="true" stored="true" default="NOW" multiValued="false"/>
+    <copyField source="*_s" dest="*_ssim"/>
+    <copyField source="*_t" dest="*_tsim"/>
+  
+    <!-- Temporary Dynamic Fields that mimic the Argo schema to allow Purl-Fetcher fields to Match Dor-Fetcher fields -->
+     <!-- Following the upgrade of Argo-prod these will be delated and _ssim/_tsim will be used -->
+    <dynamicField name="*_s"  type="string"  indexed="true"  stored="true" multiValued="true"/>
```

Comment:
I can do that if you want, no problem.  Although if in the future the code were to depend on some other _s, I would need that _s in both cores since 1:1 functionality is needed.

Basically when I add in something here:

https://github.com/sul-dlss/dor-fetcher-service/blob/master/config/solr_terms.yml (ex: collection_field), it needs to be in both cores.  Since it is already in Argo as _s (until Argo is updated), it ends up in purl as _s.  Net result would be I'd just throw in another pull request for that field.  If you prefer that over dynamic though I could do it, might just result in more pulls as I realize other fields are needed.